### ISSUE TITLE
Issue on window writing a new mail

### ISSUE THREAD

**Issue by londonuk371** [spymastermatt/thunderbird-monterail#92]
The montreil look for Thunderbird is very good but when I want to write a new mail buttons on "Send" line are written black with a dark background. How to fix that?
My options in chrome/userChrome.css are:
/******************************************************************/
/*********************** SELECT THEME HERE ************************/
@import "themes/monterail.css";
/******************************************************************/
/******************************************************************/

/* Uncommment to use the dark folder pane style when using the system theme with dark menus */
@import "tweaks/dark-folder-pane.css";

/* Select icon theme (leave commented to use default for the selected theme)*/
@import "icons/lightIcons.css"; /*For light Folder Tree and Message List*/

Thanks.
![thunderbird-montreil](https://user-images.githubusercontent.com/22617037/35092813-d79216ea-fc40-11e7-8f8c-c6ab2d04fdcb.jpg)

**Comment by sklaes** [comment]
I do have the same issue

**Comment by conema** [comment]
Hi @londonuk371 @sklaes, are you using an updated version of this theme? That problem should be resolved by #69 

May you post a screenshot of the menu bar? (press _alt_ in the new mail window)

**Comment by londonuk371** [comment]
@conema I downloaded the last version from https://github.com/spymastermatt/thunderbird-monterail with the same result.
I get this with alt:
![thumderbird-monterail](https://user-images.githubusercontent.com/22617037/35919982-d473fa10-0c16-11e8-9bf9-27f569a3c20f.jpg)
I tried to modify the file monterail.css at line 39:
  --monterail-text-color: #353535;
to 
  --monterail-text-color: #f8f8f8;
Then I get:
![thumderbird-monterail-2](https://user-images.githubusercontent.com/22617037/35920027-f4400e10-0c16-11e8-8cfd-10fc75da43e6.jpg)
We can't see anymore text right to icons and emails subjects.

We should be able to set 2 colors, one for the menu and emails subjects, another one for the menu with the green background.

Cheers,
Vincent.

**Comment by conema** [comment]
Hi @londonuk371, this seems very strange, I cannot reproduce this issue. Which version of Thunderbird are you using? 

However, for resolve your issue try to add this code at the end of themes/monterail.css (this is only for the alt menu, let me know if it works):
`#navigation-toolbox .menubar-text{ color:  #f8f8f8 !important; }`

**Comment by londonuk371** [comment]
@conema 
Ok, almost done. What I had to comment and add in the file themes/monterail.css:
```
#navigation-toolbox .splitmenu-menuitem label, menucaption label, .chromeclass-toolbar menu label, .chromeclass-toolbar menuitem label, .chromeclass-toolbar .splitmenu-menuitem label, .chromeclass-toolbar menucaption label {
    /*color: var(--monterail-text-color) !important;*/
    color: #f8f8f8 !important;
}

```

```
.toolbarbutton-1, .toolbarbutton-menubutton-button, #composeToolbar2{
     /*color: var(--dark-text-color) !important;*/
     color: #f8f8f8 !important;
}
```
I miss only the icons in white on writing a new mail in the popup, which are grey (almost invisible):
![thumderbird-monterail-3](https://user-images.githubusercontent.com/22617037/35931139-1d9cf062-0c34-11e8-8987-d455a1efa554.jpg)

**Comment by conema** [comment]
@londonuk371 For icons:

- The rows from 34 to 39 included, should be commented in userChrome.css
- monterail.css should include _../icons/lightDarkIcons.css_ (usually in row 27)

However there should not be these problems. Which version of Thunderbird are you using?

**Comment by sklaes** [comment]
Replacing the following at the end of monterail.css:
`
 .toolbarbutton-1, .toolbarbutton-menubutton-button, #composeToolbar2{ color: var(--dark-text-color) !important; }
`
with 
`
.toolbarbutton-1, .toolbarbutton-menubutton-button, #composeToolbar2{ color: #f8f8f8 !important; }
`
fixed the main issue
![image](https://user-images.githubusercontent.com/13706940/35932898-58df7366-0c39-11e8-85a8-e1c3b2d18952.png)

using latest monterail and Thunderbird 52.6.0

my menu bar is still grey, with a black font

**Comment by londonuk371** [comment]
@conema Yes, sorry forgot to reply about the version of Thunderbird. My version is 52.6.0 (32 bits). Might be the last as I do all automatic updates.

@sklaes If you have this trouble by clicking on Addresses to see all collected addresses, there are icons but no text right to icons: 
![thumderbird-monterail-4](https://user-images.githubusercontent.com/22617037/35996453-e04cbb82-0d16-11e8-819e-57cb96e76c20.jpg)
I modified the last css block to 2 blocks:
```
.toolbarbutton-1, .toolbarbutton-menubutton-button {
    color: var(--dark-text-color) !important;
}

#composeToolbar2 {
    color: #f8f8f8 !important;
}
```
